risc0/cargo-risczero/README.md

```diff
@@ -0,0 +1,32 @@
+# Cargo-risczero
+
+Cargo extension to help create, manage, and test [risc0](https://github.com/risc0/risc0) projects.
```

Comment:
Our style guidance is "RISC Zero" for most general use cases ("develop RISC Zero projects"), and "`risc0`" only for the literal string needed in code and commands ("enter the `risc0` directory"). As a rule of thumb, use "`risc0`" when you're using monospace/backticks, and use "RISC Zero" when you are not formatting the text as code.
```suggestion
Cargo extension to help create, manage, and test [RISC Zero](https://github.com/risc0/risc0) projects.
```